analyzers/FacsCalibur/manifest.mf

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0

```

Comment:
This file should not be added to version control. It belongs with the IDE (Netbeans